_posts/dotnet/2020-02-08-setting-up-docfx-on-github-pages.md

```diff
@@ -5,23 +5,24 @@ tags: [github-actions, c#, dotnet]
 ---
 
 {% include JB/setup %}
-Having good documentation is important to the success of any kind of software project. For open source projects libraries it: reduces the amount of questions the maintainers needs to respond to, increases adoption and makes it is easier for others to contribute to the project.
+Having clear documentation is important to the success of any kind of software project. For open source projects libraries, it reduces the amount of questions the maintainers need to respond to, increases adoption and makes it easier for others to contribute to the project.
 
-The guide details how you can create a free documentation website hosted on github pages using [Microsoft's open source DocFX](https://dotnet.github.io/docfx/index.html) library for a dotnet project (although it could easily be modified to support typescript/javascript libraries too. This tool will generate a static searchable website based on your markdown files and public exposed classes and members.
+This guide details how you can create a free documentation website hosted on github pages using [Microsoft's open source DocFX](https://dotnet.github.io/docfx/index.html) library for a dotnet project although it could easily be modified to support typescript/javascript libraries too. This tool generates a static searchable website based on your markdown files and public exposed classes and members.
+<!-- 'Public exposed classes and members sounds weird and I'm unsure what you're trying to express. -->
```

Comment:
When you add comments to your classes and methods, automatic tools can read this metadata and generate html out of them like this 
https://www.npgsql.org/doc/api/Npgsql.html

This is called the API Reference or API Documentation.

Checking this is how you can easily learn about all the functionality of a library.

Reply:
This tool generates a static and searchable website based on your public classes and methods and markdown files.

I don't even know the correct grammar for that. 

Public exposed sounds weird, public makes more sense to me. I also didn't know a member was a method

Reply:
How about `This tool generates API reference documentation based on your publicly exposed members such as classes and methods and their comments.`??

Reply:
Sounds fine to me